### PR TITLE
Add Ephemerides to Dockerfile

### PR DESCRIPTION
**Pull request by Rodrigo-Tenorio**
Solve #74 including steps to download ephemerides during docker build.

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -1,7 +1,25 @@
 FROM python:3.8-buster
 
 WORKDIR /pyfstat
+
+# Ephemerides arguments
+## Put ephemerides in this folder
+ARG EPHEMERIDES_FOLDER=ephemerides/
```
Comment (Rodrigo-Tenorio) [inline Dockerfile]: Actually, I'm not sure of this relative path:

WORKDIR ensures that we are referring to /pyfstat/ephemerides, but once spawned, LALPULSAR_DATADIR will be "ephemerides/" only (relative path)

**Comment by dbkeitel** [comment]
I guess you could temporarily add the branch to the .yml, let it build the image once, then when it looks correct, drop that change again from the commit before merging...?

**Comment by Rodrigo-Tenorio** [comment]
GitHub does not let you remove packages from a public repository (e.g. look at the vDummyTag), so I'd prefer to keep the docker-clutter low. 

I'm building locally, using get_ephemeris_files to check whether ephemeris are there.

**Comment by Rodrigo-Tenorio** [comment]
@dbkeitel that should be it.  

- `ARG` values can be set at build time (in case anyone wants to try some fancy ephemerides). 
- `ENV` will set environmental variables at run time (i.e where are the ephemerides?)

**Pull request merged**